src/code_0800A868.c

```diff
@@ -31,3 +31,23 @@ s32 sub_0800A868(u8 playerId, struct Object *obj)
 	
     return var2;
 }
+
+void sub_0800A8BC(void)
+{
+    gUnk_03000004 = 0;
```

Comment:
Does `*(u32 *)(IWRAM + 4) = 0` match? We've seen the pattern elsewhere so the symbol probably doesn't exist